pythran/pythonic/include/utils/shared_ref.hpp

```diff
@@ -8,7 +8,6 @@
 #include <atomic>
 #endif
 #ifdef ENABLE_PYTHON_MODULE
-#include <boost/python/object.hpp>
```

Comment:
On Mon, Oct 05, 2015 at 12:50:02PM -0700, pbrunet wrote:
> > @@ -8,7 +8,6 @@
> >  #include <atomic>
> >  #endif
> >  #ifdef ENABLE_PYTHON_MODULE
> > -#include <boost/python/object.hpp>
> 
> ifdef are not needed anymore

They are needed for the (unused) pure C++ mode, where you d'ont want any
Python C API stuff